snuba/subscriptions/subscription.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime, timedelta
```

Comment:
Not totally happy with the name of this file. These are kind of similar to the mediators that ecosystem uses, potentially we could call them something like that.

Reply:
I don't have a great idea either, I think this works for now though.